easybuild/easyconfigs/s/Seqtk/Seqtk-1.2-goolf-1.7.20-r94.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'Seqtk'
```

Comment:
we already have easyconfigs that use `seqtk` as name

the GitHub page doesn't have a strong suggestion it should be `Seqtk` instead, so stick to `seqtk`?